src/snowflake/cli/plugins/nativeapp/v2_conversions/v2_to_v1_decorator.py

```diff
@@ -25,12 +26,25 @@
 from snowflake.cli.api.project.schemas.entities.application_package_entity import (
     ApplicationPackageEntity,
 )
+from snowflake.cli.api.project.schemas.native_app.path_mapping import PathMapping
 from snowflake.cli.api.project.schemas.project_definition import (
     DefinitionV11,
     DefinitionV20,
 )
 
 
+def _convert_v2_artifact_to_v1_dict(
+    v2_artifact: Union[PathMapping, Path]
+) -> Union[Dict, str]:
+    if type(v2_artifact) == PathMapping:
```

Comment:
the idiomatic python way is to use `isinstance` (which handles subclasses)
```suggestion
    if isinstance(v2_artifact, PathMapping):
```